dubbo-demo/dubbo-demo-spring-boot/dubbo-demo-spring-boot-provider/pom.xml

```diff
@@ -133,6 +132,26 @@
             <artifactId>log4j</artifactId>
         </dependency>
 
+        <!-- Observability -->
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-metrics-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-tracing-bridge-otel</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-exporter-zipkin</artifactId>
+            <version>1.20.0</version>
```

Comment:
Version should be managed in `dubbo-dependencies-bom`

Reply:
I removed that to simplify the example